.gitignore

```diff
@@ -1,2 +1 @@
-.idea
-_build
+.DS_Store
```

Comment:
this needs a carriage return (put CR at end of files)

Also, this .gitignore shouldn't be in the db directory.  I think you will find one already in the top directory that this entry should be added to.

Reply:
so should I delete this .gitignore file? I'm a little confused as how to get the pre-existing .gitignore file and add to that.